rsocket-ipc-jackson/src/main/java/io/rsocket/ipc/marshallers/Cbor.java

```diff
@@ -0,0 +1,56 @@
+package io.rsocket.ipc.marshallers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.cbor.CBORFactory;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.buffer.ByteBufOutputStream;
+import io.rsocket.ipc.Marshaller;
+import io.rsocket.ipc.Unmarshaller;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import reactor.core.Exceptions;
+
+public class Cbor {
+  private final ObjectMapper mapper;
+
+  private Cbor() {
+    CBORFactory f = new CBORFactory();
+    mapper = new ObjectMapper(f);
+  }
+
+  private static class Lazy {
+    private static Cbor INSTANCE = new Cbor();
+  }
+
+  public static Cbor getInstance() {
+    return Lazy.INSTANCE;
+  }
+
+  @SuppressWarnings("unused")
+  public static <T> Marshaller<T> marshaller(Class<T> clazz) {
+    return t -> {
+      ByteBuf byteBuf = ByteBufAllocator.DEFAULT.buffer();
```

Comment:
Do you expect to make the allocator configurable in future?

Reply:
I could - these will get more polish with use i think